docs/src/getstarted/hello-world.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Hello World Quickstart Guide"
+description: "This quickstart guide will demonstrate how to setup, build, and deploy your first Solana program in your browser with Solana Playground."
+keywords: "playground, solana pg, on chain, rust, native program, tutorial, intro to solana development, blockchain developer, blockchain tutorial, web3 developer"
+---
+
+For this quickstart guide, we will use [Solana Playground](https://beta.solpg.io), a browser the based IDE, to develop and deploy our Solana program. To use it, you do **NOT** have to install any software on your computer. Simply open Solana Playground in your browser of choice, and you are read to write and deploy Solana programs.
```

Comment:
Do we still call them "quickstart guides" if they're under `getting started` now?